akka-stream-typed/src/test/scala/docs/akka/stream/typed/ActorSourceSinkExample.scala

```diff
@@ -55,16 +58,20 @@ object ActorSourceSinkExample {
     case object Complete extends Protocol
     case class Fail(ex: Throwable) extends Protocol
 
-    val actor: ActorRef[Protocol] = ???
+    val actor: ActorRef[Protocol] = targetActor()
 
     val sink: Sink[Protocol, NotUsed] =
       ActorSink.actorRef[Protocol](ref = actor, onCompleteMessage = Complete, onFailureMessage = Fail.apply)
 
     Source.single(Message("msg1")).runWith(sink)
     // #actor-sink-ref
+
   }
 
   {
+
+    def targetActor(): ActorRef[Protocol] = ???
```

Comment:
is this really a side effect needing `def targetActor(): ActorRef[Protocol]` versus def targetActor: ActorRef[Protocol]?